src/http/client.rs

```diff
@@ -248,7 +248,16 @@ impl EspHttpConnection {
 
         self.request_content_len = content_len.unwrap_or(0);
```

Comment:
Assuming you'll change `request_content_len` to `i64`, as suggested below, just do here `unwrap_or(-1)`